docs/src/app/components/pages/components/Icons/FontREADME.md

```diff
@@ -0,0 +1,11 @@
+## Font Icons
+This component will render any [icon](https://www.google.com/design/spec/style/icons.html#) defined in any style sheets included in your project. We are using [Google's Material Design Icons](https://github.com/google/material-design-icons) for our documentation site along with some custom icons. You can use sites like IcoMoon for generating custom font files. To use FontIcons, add your stylesheet to your project and reference the icon's className in the "className" prop.
```

Comment:
Can you split the line. It's going to be far too long in my editor.
We are lucky here. Github is wrapping the markdown but not the source code.